app/internal/templatefn/fn.go

```diff
@@ -37,6 +39,15 @@ var All = map[string]interface{}{
 	"warning":        warning,
 }
 
+var LpaPropertyToTransKey = map[string]string{
+	"pfa":                   "lpaTypePfa",
+	"jointly":               "jointly",
+	"jointly-and-severally": "jointlyAndSeverally",
+	"mixed":                 "jointlyAndSeverallyMixed",
+	"when-registered":       "asSoonAsItsRegistered",
+	"when-capacity-lost":    "onlyWhenIDoNotHaveMentalCapacity",
+}
```

Comment:
why not just make these normal translations?

Reply:
Mainly because my eye was twitching at the mix of styles in the JSON keys! Having to write that I now realise that's a pretty lame reason to add more complexity so I'll suck up my OCD and add some kebab case keys in the translation files.